packages/client/components/AddTeamMemberModal.tsx

```diff
@@ -34,12 +35,49 @@ interface State {
   rawInvitees: string
 }
 
+const INVITE_DIALOG_WITH_ILLUS = 864
```

Comment:
should we move this into the constEnums/Breakpoints const enum? My thought is that when we build other modals, we'll probably want to use this breakpoint as a reference.

Reply:
I’d like to keep it local until we repeat the pattern.